snap/snapcraft.yaml

```diff
@@ -946,12 +946,33 @@ parts:
       patches: openresty-kong-patches
     stage: [openresty-kong-patches]
     prime: [-*]
+  lua-kong-nginx-module:
+    source: https://github.com/Kong/lua-kong-nginx-module.git
+    source-tag: 0.0.4
+    plugin: nil
+    override-build: |
+      # install the static lualib dir into the final snap, the compiled parts
+      # will be installed into the snap as part of the OpenResty part build
+      mkdir -p $SNAPCRAFT_PART_INSTALL/lualib/resty/kong
+      cp -r $SNAPCRAFT_PART_SRC/lualib/resty/kong/tls.lua $SNAPCRAFT_PART_INSTALL/lualib/resty/kong/tls.lua
+
+      # copy the necessary soruce files from here into a module specific dir in
+      # $SNAPCRAFT_STAGE because some of these files such as config conflict 
+      # with files that Kong generates/needs, so we want these to be in their
+      # own dir for OpenResty to compile with
+      mkdir -p $SNAPCRAFT_STAGE/lua-kong-nginx-module
+      for f in lualib src config Makefile; do
+        cp -r "$SNAPCRAFT_PART_SRC/$f" "$SNAPCRAFT_STAGE/lua-kong-nginx-module/$f"
+      done
+      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/share/lua-kong-nginx-module
```

Comment:
shouldn't there be a /doc in the path under /usr/share?

Reply:
oh yeah I suppose there should be